### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by martinmona**
## Summary
- Completed responsive book details

## Screenshot
image

## Trello Card
https://trello.com/c/3NUM0wu2/34-responsive

### CODE REVIEW

File: Maquetado/home.html [hunk 1]
```diff
@@ -6,12 +6,13 @@
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
   <link rel="stylesheet" type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
```
Comment (henryzarza) [inline Maquetado/home.html]: Move this to `application.scss` file

File: Maquetado/index.html [hunk 2]
```diff
@@ -6,23 +6,29 @@
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
   <link rel="stylesheet" type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
```
Comment (henryzarza) [inline Maquetado/index.html]: The same here

File: Maquetado/login.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Login</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
```
Comment (henryzarza) [inline Maquetado/login.html]: ☝️

File: Maquetado/scss/details.scss [hunk 4]
```diff
@@ -125,3 +146,59 @@
   line-height: #{$line-height-base -2};
   margin-top: 5px;
 }
+
+@media only screen and (max-width: 1024px) {
```
Comment (henryzarza) [inline Maquetado/scss/details.scss]: Move this value (1024px) to variables file

File: Maquetado/scss/details.scss [hunk 5]
```diff
@@ -125,3 +146,59 @@
...
+  .book {
+    align-items: center;
+    flex-direction: column;
+    height: 100%;
+    max-height: 700px;
+    max-width: 350px;
+    width: 100%;
```
Comment (henryzarza) [inline Maquetado/scss/details.scss]: Delete this

File: Maquetado/scss/details.scss [hunk 6]
```diff
@@ -125,3 +146,59 @@
...
+  }
+
+  .book-img {
+    height: 100%;
```
Comment (henryzarza) [inline Maquetado/scss/details.scss]: Delete this

File: Maquetado/scss/details.scss [hunk 7]
```diff
@@ -125,3 +146,59 @@
...
+    max-height: 300px;
+    width: 210px;
+  }
+
+  .book-badge {
+    left: -85px;
+    position: absolute;
```
Comment (henryzarza) [inline Maquetado/scss/details.scss]: Delete this

File: Maquetado/scss/details.scss [hunk 8]
```diff
@@ -125,3 +146,59 @@
...
+    top: 20px;
```
Comment (henryzarza) [inline Maquetado/scss/details.scss]: ☝️

File: Maquetado/scss/details.scss [hunk 9]
```diff
@@ -125,3 +146,59 @@
...
+    transform: rotate(20deg) scale(calc(#{$badge-scale*$badge-media-scale}));
+    transform-origin: top right;
```
Comment (henryzarza) [inline Maquetado/scss/details.scss]: Delete this

File: Maquetado/scss/details.scss (hunk 5 above)
Comment (manueltuero) [reply Maquetado/scss/details.scss]: He can delete this `width: 100%;` because he's using flex?

File: Maquetado/scss/details.scss (hunk 6 above)
Comment (manueltuero) [reply Maquetado/scss/details.scss]: Yes, please delete this `height: 100%;`.

File: Maquetado/scss/details.scss [hunk 10]
```diff
@@ -15,16 +15,37 @@
...
+}
+
+.container {
   background-color: $wild-sand;
   height: 100%;
   width: 100%;
 }
 
+.container-book {
+  display: flex;
```
Comment (manueltuero) [inline Maquetado/scss/details.scss]: Using `flex` here is necessary?

Comment (martinmona) [reply Maquetado/scss/details.scss]: If i don't use it, then the elements are not aligned. Instead, is one above the other.

**Pull request merged**